src/ncp/ncp_base.cpp

```diff
@@ -869,7 +869,8 @@ void NcpBase::CommandHandler_NOOP(uint8_t header, unsigned int command, const ui
 void NcpBase::CommandHandler_RESET(uint8_t header, unsigned int command, const uint8_t *arg_ptr, uint16_t arg_len)
 {
     // TODO: Figure out how to actually perform a reset.
-    otInit();
+    otDisable();
+    otEnable();
```

Comment:
Unrelated: We should probably add a `otPlatReset()` method, intended to cause the platform to perform a software reset.